app/models/journey.rb

```diff
@@ -46,6 +46,15 @@ def keywords_search
     join_search([to_search_str(keywords), steps_search(:keywords)])
   end
 
+  def reposition
+    steps.each_with_index do |step, index|
+      position_by_index = (index + 1) * 2
```

Comment:
Preco tam nie je len `index + 1` ?

Reply:
To je len napad, aby tam bol viditelny priestor na pridanie kroku medzi uz dva existujuce. Po reposition budu pozicie 2, 4, 6, takze novy krok sa da lahko pridat na zaciatok, medzi, alebo na koniec.